packages/react/src/components/Button/Button.test.tsx

```diff
@@ -83,60 +59,57 @@ describe('Button', () => {
         }}
       />,
     );
-    const tree = component.toJSON();
-    expect(tree).toMatchSnapshot();
+    expect(container).toMatchSnapshot();
   });
 
   it('renders CompoundButton correctly', () => {
-    const component = create(
+    const { container } = render(
       <CompoundButton secondaryText="You can create a new account here.">Create account</CompoundButton>,
     );
-    const tree = component.toJSON();
-    expect(tree).toMatchSnapshot();
+    expect(container).toMatchSnapshot();
   });
 
   it('renders IconButton correctly', () => {
-    const component = create(<IconButton iconProps={{ iconName: 'Emoji2' }} title="Emoji" ariaLabel="Emoji" />);
-    const tree = component.toJSON();
-    expect(tree).toMatchSnapshot();
+    const { container } = render(<IconButton iconProps={{ iconName: 'Emoji2' }} title="Emoji" ariaLabel="Emoji" />);
+    expect(container).toMatchSnapshot();
   });
 
   describe('DefaultButton', () => {
     it('can render without an onClick.', () => {
-      const button = render(<DefaultButton>Hello</DefaultButton>);
+      const { getByRole } = render(<DefaultButton>Hello</DefaultButton>);
 
-      expect(button.tagName).toEqual('BUTTON');
+      expect(getByRole('button').tagName).toEqual('BUTTON');
```

Comment:
This test is a bit weird...I almost wonder if it would be better for this specific basic test (and the one for link buttons) to check `container.firstElementChild.tagName`?

Reply:
Yeah that makes more sense! I'll make the change